modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/ionchannelmechanismrelease/v0.1.2.json

```diff
@@ -0,0 +1,44 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/ionchannelmechanismrelease/v0.1.2/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/nexus/core/distribution/v0.1.0",
+    "{{base}}/schemas/neurosciencegraph/simulation/modelrelease/v0.1.2",
+    "{{base}}/schemas/neurosciencegraph/simulation/subcellularmodel/v0.1.2"
+  ],
+  "shapes": [
+    {
+      "@id": "this:IonChannelMechanismReleaseShape",
+      "@type": "sh:NodeShape",
+      "label": "This shape is for...",
+      "targetClass": "nsg:IonChannelMechanismRelease",
+      "nodekind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/simulation/modelrelease/v0.1.1/shapes/ModelReleaseShape"
+        },
+        {
+          "property": [
+            {
+              "path": "nsg:distribution",
+              "name": "Ion channel release data location",
+              "description": "Ion channel release data location.",
+              "node": "{{base}}/schemas/nexus/core/distribution/v0.1.0/shapes/DistributionShape"
```

Comment:
I would suggest to constrain the media type here. I think only mod files are allowed. Am I right ?

Reply:
This will be link to git repo/hash/tag.

Reply:
You will use accessURL I guess.

Reply:
yep